tests/phpunit/tests/html-api/wpHtmlProcessor.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Unit tests covering WP_HTML_Processor functionality.
+ *
+ * @package WordPress
+ * @subpackage HTML-API
+ */
+
+/**
+ * @group html-api
```

Comment:
```suggestion
 * @subpackage HTML-API
 *
 * @group html-api
```
Let's merge these docblocks as was done in [this ticket](https://core.trac.wordpress.org/ticket/57099).

🔢 Applies elsewhere in the PR.

Reply:
updated in 9ad657f438

would you mind checking that this meets the requirements?

also, on an unrelated note, with `@coversDefaultClass` do we need to repeat `WP_HTML_Processor::next_tag` on all the individual `@covers` cases or can we just use `::next_tag`?